hack/bin/integration-test.sh

```diff
@@ -1,9 +1,79 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 NAMESPACE=${NAMESPACE:-test-integration}
 
 echo "Running integration tests on wire-server"
 
 CHART=wire-server
-helm test --logs -n "${NAMESPACE}" "${NAMESPACE}-${CHART}" --timeout 900s
+tests=(cargohold gundeck federator spar brig galley)
+
+summary() {
+    echo "==============="
+    echo "=== summary ==="
+    echo "==============="
+    printf '%s\n' "${tests[@]}" | parallel echo "=== tail {}: ===" ';' tail -2 logs-{}
+
+    for t in "${tests[@]}"; do
+        x=$(cat "stat-$t")
+        if ((x > 0)); then
+            echo "$t-integration FAILED ❌. pfff..."
+        else
+            echo "$t-integration passed ✅."
+        fi
+    done
+}
+
+# Run tests in parallel using GNU parallel (see https://www.gnu.org/software/parallel/)
+# The below commands are a little convoluted, but we wish to:
+# - run integration tests. If they fail, keep track of this, but still go and get logs, so we see what failed
+# - run all tests. Perhaps multiple flaky tests in multiple services exist, if so, we wish to see all problems
+touch ~/.parallel/will-cite
+printf '%s\n' "${tests[@]}" | parallel echo "Running helm tests for {}..."
+printf '%s\n' "${tests[@]}" | parallel -P 6 \
+    helm test -n "${NAMESPACE}" "${NAMESPACE}-${CHART}" --timeout 900s --filter name="${NAMESPACE}-${CHART}-{}-integration" '> logs-{};' \
```

Comment:
This will most likely not work with changes in #3037 as the filter will only match the pods and not actually install other components.

Reply:
Let's try: https://github.com/wireapp/wire-server/pull/3042